RPIPs/RPIP-43.md

```diff
@@ -8,37 +8,95 @@ status: Draft
-- Legacy RPL staking (ie staking RPL to a node rather than to a megapool) SHALL be disabled
-- A user MAY set any amount of RPL "Staked" to a megapool to "Unstaking"
-- Once RPL has been "Unstaking" for `unstaking_period` days, it can be withdrawn (this setting is defined in [RPIP-30](RPIP-30.md))
-- All RPL that was "Unstaking" MUST be withdrawn at one time
-- There SHALL only be a single time stored for "Unstaking" RPL; this means that if a user sets
-  additional RPL to "Unstaking" after some RPL is already "Unstaking", they must wait for
-  `unstaking_period` before any is available to withdraw
-  - All RPL that was already "Unstaking" for `unstaking_period` SHOULD be withdrawn when a user
-    sets additional RPL to "Unstaking"
+
+### RPL Staking
+
+This RPIP includes a rework of the treatment of RPL staked on Rocket Pool
+nodes. We refer to RPL staked for a node prior to the implementation of this
+RPIP as "legacy staked RPL", and to RPL staked according to the rules described
+in this RPIP as, simply, "staked RPL", or as "megapool staked RPL" if otherwise
+ambiguous.
+
+Megapool staked RPL (in the protocol, ie, before being withdrawn) can be in one
+of two states:
+- Staked: the RPL has been deposited and staked (locked) in the protocol, for a
+          node with a megapool deployed
+- Unstaking: the RPL has been marked as intended to be withdrawn
+
+Rules specifying the movement of staked RPL are as follows:
+
+- Legacy RPL staking (ie, increasing a node's legacy staked RPL balance) SHALL
+  be disabled
+- There is no staked RPL requirement for adding validators to a megapool
+- A Node Operator MAY change the state of any amount of staked RPL associated
+  with their node from Staked to Unstaking at any time
+    - The time at which Staked RPL was last changed to Unstaking RPL is called the
+      "last unstake time" for the node
+- Only Unstaking RPL can be withdrawn from the protocol, and only if the last
+  unstake time for the node is at least `unstaking_period` (this parameter is
+  defined in [RPIP-30](RPIP-30.md))
+- If any Unstaking RPL is withdrawn for a node, all Unstaking RPL for that node
+  MUST be withdrawn at once
+- If the last unstake time is at least `unstaking_period` when additional RPL
+  is having its state changed Unstaking, the RPL already in the Unstaking state
+  SHOULD be withdrawn first
 
 ### Vote Eligibility
-- All nodes are eligible to receive vote power.
-- For nodes eligible to receive vote power, total RPL staked (to either a megapool or the node) SHALL contribute to vote power up to 150% of the value of total bonded ETH (from both megapools and legacy minipools).
+
+- All nodes are eligible to receive vote power
+- Total RPL staked (to either a megapool or the node) SHALL contribute to vote power up to 150% of the value of total bonded ETH (from both megapools and legacy minipools)
```

Comment:
```
- All nodes are eligible to receive vote power.
- For nodes eligible to receive vote power, total RPL staked (to either a megapool or the node) SHALL contribute to vote power up to 150% of the value of total bonded ETH (from both megapools and legacy minipools).
```
The wording was to explicitly mirror https://rpips.rocketpool.net/RPIPs/RPIP-4#eligibility. I think it should continue to do so.

Reply:
Ugh I wish I'd caught it in RPIP 4 then lol. OK.